### DISCUSSION TITLE
Vue 3 event validation: returned value is ignored - why?

### DISCUSSION THREAD

**Discussion by iliubinskii** [vuejs/docs#1355] [Q&A] [answered]
Vue 3 documentations says that event validator should return boolean value (see screenshot below).
But this value is ignored.
Is it bug?
If not then how can I see the effect of this returned value?

https://v3.vuejs.org/guide/component-custom-events.html#defining-custom-events
![Screen](https://user-images.githubusercontent.com/70687432/144007285-906c40d2-445a-4fed-b02e-0d7e17659759.jpg)

**Comment by bencodezen** [comment]
Hi @ilyub! It should show up in your console when you're developing. It's not something that will break on production builds based on my understanding. If you're still not seeing it, a small reproduction for us to take a look at would be greatly appreciated!

**Comment by iliubinskii** [comment, marked as answer]
Hi, it turned out that I have errors enabled and warnings disabled in my browser output filter.
After enabling warnings channel I see Vue warning.
Thx for the help.
